### PR TITLE
Replace axios with native fetch

### PR DESCRIPTION
**Pull request by mgwalker**
Axios doesn't really provide us much benefit over native fetch, so yoink it out and that's one less dependency to worry about!

Will update dev documentation just prior to merging.

---

Checklist:

- [X] Code has been formatted with prettier
- ~The [OAuth](https://github.com/18F/charlie/wiki/OAuthEventsAndScopes) wiki page has been updated if Charlie needs any new OAuth events or scopes~
- ~The [Environment Variables](https://github.com/18F/charlie/wiki/EnvironmentVariables) wiki page has been updated if new environment variables were introduced or existing ones changed~
- [ ] The dev wiki has been updated, e.g.:
  - ~local development processes have changed~
  - ~major development workflows have changed~
  - ~internal utilities or APIs have changed~
  - [ ] testing or deployment processes have changed
- ~If appropriate, the NIST 800-218 documentation has been updated~

### CODE REVIEW

File: src/scripts/define.test.js [hunk 1]
```diff
@@ -228,8 +227,8 @@ describe("glossary", () => {
 
       const fetcher = cache.mock.calls[0].pop();
 
-      axios.get.mockResolvedValue({
-        data: `
+      fetch.mockResolvedValue({
+        text: async () => `
```
Comment (Sgtpluck) [inline src/scripts/define.test.js:231]: ok this looked weird but i looked at the whole file in context and figured it out

File: src/scripts/dot-gov.js [hunk 2]
```diff
@@ -186,15 +185,13 @@ const selectDomainsAtRandom = (domainsArr, numToSelect) => {
...
-      }),
-    })
+  fetch(`https://${domainObj[DATA_FIELDS.NAME]}`, {
+    method: "HEAD",
+    agent: new https.Agent({ rejectUnauthorized: false }),
+    signal: AbortSignal.timeout(timeout),
+  })
     .then((response) => {
-      if (response && response.status) {
+      if (response?.ok && response.status) {
```
Comment (Sgtpluck) [inline src/scripts/dot-gov.js:194]: ahh i didn't know js had a null-chaining operator!

File: src/scripts/opm_status.js [hunk 3]
```diff
@@ -38,12 +37,10 @@ module.exports = (robot) => {
       incrementStats("OPM status");
 
       try {
-        const { data, status } = await axios.get(
+        const data = await fetch(
           "https://www.opm.gov/json/operatingstatus.json",
-        );
-        if (status !== 200) {
```
Comment (Sgtpluck) [inline src/scripts/opm_status.js:44]: [question] can we get rid of this error throw because `data` is more consistent now?

**Review by Sgtpluck — APPROVED** [review]
a couple small questions but nothing blockin! nice work.

File: src/scripts/define.test.js (hunk 1 above)
Comment (mgwalker) [reply src/scripts/define.test.js:231]: I'll add a comment that it needs to be formatted like yaml so maybe it looks weird to the next person. Would that help?

File: src/scripts/opm_status.js (hunk 3 above)
Comment (mgwalker) [reply src/scripts/opm_status.js:44]: Yup

File: src/scripts/define.test.js (hunk 1 above)
Comment (Sgtpluck) [reply src/scripts/define.test.js:234]: i don't think it's necessary, it was easy enough to see when i opened the file!

**Pull request merged**